service/worker/batcher/batcher.go

```diff
@@ -81,7 +76,7 @@ func (s *Batcher) Start() error {
 		BackgroundActivityContext: ctx,
 	}
 	sdkClient := s.sdkClientFactory.GetSystemClient()
-	batchWorker := worker.New(sdkClient, taskQueueName, workerOpts)
+	batchWorker := worker.New(sdkClient, primitives.PerNSWorkerTaskQueue, workerOpts)
```

Comment:
or do we want to use `temporal-sys-batcher-taskqueue` instead of primitives.PerNSWorkerTaskQueue here https://github.com/temporalio/temporal/pull/3428/files#diff-c9a59445e489bd70346630f2773ad69e1723b2298fadbae50ff5c1a424917e1cR3760 ?